lib/src/pageObjects/adminUIListScreen.js

```diff
@@ -1,110 +1,494 @@
+								} else {
+									throw new Error('clickItemFieldValue: clickListScreenFieldValue command not defined in ' + field.name + ' field test object!');
+								}
+							} else {
+								throw new Error("clickItemFieldValue: invalid field name!");
+							}
+						} else {
+							throw new Error("clickItemFieldValue: invalid field modelTestConfig!");
+						}
+					} else {
+						throw new Error("clickItemFieldValue: invalid field config row/column!");
+					}
+				});
+			} else {
+				throw new Error("Invalid field specification!");
+			}
+			return this;
 		},
-		secondItemNameValue: {
-			locateStrategy: 'xpath',
-			selector: '//tbody/tr[2]/td/a[contains(@class, "ItemList__value ItemList__value--text")][1]',
+
+		/**
+		 * Asserts that the specified item delete icon is visible.
+		 *
+		 * @param {Object} icons The delete icons specification.
+		 * @param {number} icon.row The one-based item row number of the delete icon in the list screen.
+		 * @param {number} icon.column The one-based column number of the delete icon in the list screen.
+		 */
+		assertItemDeleteIconVisible: function (icons) {
+			var browser = this;
+			if (icons) {
+				icons.forEach(function (icon) {
+					browser.expect.element(getItemDeleteIconSelector(icon)).to.be.visible;
+				});
+			} else {
+				throw new Error("Invalid icons specification!");
+			}
+			return this;
 		},
-	},
-	commands: [{
-		deleteItem: function (selector) {
-			return this
-				.click(selector);
+
+		/**
+		 * Click delete item icons.
+		 *
+		 * @param {Object} icons The delete item icons specification.
+		 * @param {number} icon.row The one-based item row number of the delete icon to click in the list screen.
+		 * @param {number} icon.column The one-based item column number of the delete icon to click in the list screen.
+		 */
+		clickDeleteItemIcon: function (icons) {
+			var browser = this;
+			if (icons) {
+				icons.forEach(function (icon) {
+					browser.click(getItemDeleteIconSelector(icon));
+				});
+			} else {
+				throw new Error("Invalid icons specification!");
+			}
+			return this;
 		},
-		createFirstItem: function () {
-			return this
-				.click('@createFirstItemButton');
+
+		/**
+		 * Asserts that the Create Item button is visible in the list screen.
+		 */
+		assertCreateItemButtonVisible: function () {
+			return this.expect.element('@createItemButton').to.be.visible;
 		},
-		navigateToFirstItem: function () {
+
+		/**
+		 * Clicks the Create Item button.
+		 */
+		clickCreateItemButton: function () {
 			return this
-				.click('@firstItemLink');
+				.click('@createItemButton');
+		},
+
+		/**
+		 * Asserts the No Items Found Text is visible in the list screen.
+		 */
+		assertNoItemsFoundTextVisible: function () {
+			return this.expect.element('@noItemsFoundText').to.be.visible;
 		},
+
+		/**
+		 * Asserts the No Items Found Text value in the list screen matches the specified value.
```

Comment:
Is the "No Items Found" Text not always "No Items Found"? Why the need for this command?